client/src/components/Profile/ContactsCard.tsx

```diff
@@ -100,6 +100,13 @@ class ContactsCard extends React.Component<Props> {
         key: 'linkedIn',
       },
     ];
+    if (epamEmail) {
+      contacts.splice(0, 0, {
+        name: 'EPAM E-mail',
+        value: epamEmail,
+        key: 'epamEmail',
+      });
```

Comment:
Please consider to change it to `push`

```suggestion
      contacts.push({
        name: 'EPAM E-mail',
        value: epamEmail,
        key: 'epamEmail',
      });
```

Reply:
We cannot use push, because then it will not be in the test where we expect it.

Reply:
sorry, I don't get it. Could you please show where it will not be?

Reply:
i just checked out the branch, changed it and seems it works

Reply:
Thanks, fixed.